be/src/exec/avro_scanner.cpp

```diff
@@ -144,7 +144,7 @@ Status AvroScanner::open() {
         }
 
         serdes_conf_t* sconf =
-                serdes_conf_new(NULL, 0, "schema.registry.url", confluent_schema_registry_url.c_str(), NULL);
+                serdes_conf_new(nullptr, 0, "schema.registry.url", confluent_schema_registry_url.c_str(), NULL);
```

Comment:
NULL -> nullptr?

Reply:
This is just the suggested fix by clang-tidy tool. not necessary to cover all the occurrence of NULL -> nullptr.

The goal is, after this PR, all the source code should be covered by the clang-tidy, and can be properly checked with new issues if any.